### DISCUSSION TITLE
Tengo problemas para contactar soporté

### DISCUSSION THREAD

**Discussion by Chicook** [community/community#112464] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hola un saludo desde España., desde hace unas semanas., estoy teniendo problemas., para recuperar mi cuenta., deshabilitada., y soporte técnico no me responde para poder recuperar la cuenta. ¿Que me recomendáis?

**Comment by AbrahamRdzSus** [comment]
te resolvieron?

**Comment by v-coder247** [comment]
Hi @Chicook, thanks for participating!


Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 


We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
